app/steps/ui/newstartapply/template.html

```diff
@@ -15,25 +15,16 @@
   <div class="grid-row">
     <div class="column-two-thirds">
       <h1 class="form-title heading-large">{{content.header}}</h1>
-      <p>{{content.introduction | safe}}</p>
+      <p class="font-large">{{content.introduction | safe}}</p>
```

Comment:
This text looks a little big, I think we should be using `font-medium` here rather than `font-large`.